### PR TITLE
Setup hello react

### PR DESCRIPTION
**Pull request by bernabasy**
Hello reviewer 💯  Inthise Milestone I have:
- Create a new React app called 'hello-react-front-end'
- Initialize your project with Git.
- Create the App component with react-router.
- Create the Greeting component that will display a greeting. Set it up as a route in your App component.
- Create a store, an action, and a reducer that will connect to my API endpoint to get the random greeting.
- Display the random greeting in my Greeting component.

[Here is a link to back_end](https://github.com/bernabasy/hello-rails-back-end1/pull/1)

### CODE REVIEW

File: src/Redux/store/greetingReducer.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+export const fetchGreeting = createAsyncThunk('greeting', async () => {
+  const response = await axios.get('/api/v1/greetings');
```
Comment (Qoosim) [inline src/Redux/store/greetingReducer.js]: - [ ] The url used to fetch greetings from backend is not correct. It is supposed to be this :point_down: 
```
  const response = await axios.get(`http://127.0.0.1:3000/api/v1/greetings`)
     
     OR
   
   const response = await axios.get(`http://localhost:3000/api/v1/greetings`)
```

- [ ] I was able to get random greetings from your backend when I tested it. But this is what I got from the frontend while trying to run your app. You need to resolve the CORS error.

### App on Chrome
app-on-chrome

your-cors-issue

### App on Firefox
app-on-firefox

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @bernabasy, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bernabasy** [comment]
here is screen shoot, 
image

**Review by mukhammadrizooff — APPROVED** [review]
# FRONT-END
Hi @bernabasy , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️
- Correct GitFlow ✔️
- Good implementations ✔️
- Descriptive README file ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bernabasy** [comment]
thanks

**Pull request merged**